src/components/sbb-datepicker/sbb-datepicker.tsx

```diff
@@ -270,7 +254,22 @@ export class SbbDatepicker implements ComponentInterface {
     return this.format ? this.format(date) : this._dateAdapter.format(date);
   }
 
+  private _setAriaLiveMessage(date: Date): void {
+    const ariaLiveFormatter = new Intl.DateTimeFormat(`${this._currentLanguage}-CH`, {
+      weekday: 'long',
+      year: 'numeric',
+      month: 'numeric',
+      day: 'numeric',
+    });
+
+    this._statusContainer.innerHTML = date
+      ? `${i18nDateChangedTo[this._currentLanguage]} ${ariaLiveFormatter.format(date)}`
+      : i18nNoDate[this._currentLanguage];
+  }
+
   public render(): JSX.Element {
-    return <Host></Host>;
+    return (
+      <p role="status" class="visually-hidden" ref={(ref) => (this._statusContainer = ref)}></p>
```

Comment:
If host is already hidden for screen reader, it should not be needed here?
```suggestion
      <p role="status" ref={(ref) => (this._statusContainer = ref)}></p>
```